### PR TITLE
Blog for release of v1.2.1

### PR DESCRIPTION
**Pull request by daviian**
As title

### CODE REVIEW

**Comment by lunny** [comment]
LGTM

**Comment by lafriks** [comment]
LGTM

File: content/post/release-of-1.2.1.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+date: "2017-10-16T17:22:30+02:00"
+author: "daviian"
+title: "Release of 1.2.1"
+tags: ["release"]
+draft: false
+---
+
+We proudly present the bugfix release of Gitea version 1.2.1. We have merged [5](https://github.com/go-gitea/gitea/milestone/15?closed=1) pull requests to release this version. You can download one of our pre-built binaries from our [downloads page](https://dl.gitea.io/gitea/1.2.1/), you just need to select the correct platform. For further details of the installation follow our [installation guide](https://docs.gitea.io/en-us/install-from-binary/).
```
Comment (bkcsoft) [inline content/post/release-of-1.2.1.md]: > release 1.2.1 of Gitea
> We have merged 5 [pull requests](...) for this release.

File: content/post/release-of-1.2.1.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+<!--more-->
+
+## Changelog
+
+* BUGFIXES
+  * Fix PR, milestone and label functionality if issue unit is disabled (#2710) (#2714)
```
Comment (bkcsoft) [inline content/post/release-of-1.2.1.md]: The issue/PR-numbers should be links :)

**Pull request merged**